runtime/auth/aws-signing-default/common/test/aws/smithy/kotlin/runtime/auth/awssigning/DefaultSignatureCalculatorTest.kt

```diff
@@ -80,4 +82,44 @@ class DefaultSignatureCalculatorTest {
         val actual = SignatureCalculator.Default.stringToSign(canonicalRequest, config)
         assertEquals(expected, actual)
     }
+
+    private data class ChunkStringToSignTest(val signatureType: AwsSignatureType, val expectedNonSignatureHeaderHash: String)
+    @Test
+    fun testChunkStringToSign() {
+        // Test event stream signing
+        // https://docs.aws.amazon.com/transcribe/latest/dg/streaming-http2.html
+        // Adapted from: https://github.com/awslabs/smithy-rs/blob/v0.38.0/aws/rust-runtime/aws-sigv4/src/event_stream.rs#L166
+        val tests = listOf(
+            ChunkStringToSignTest(AwsSignatureType.HTTP_REQUEST_CHUNK, HashSpecification.EmptyBody.hash),
+            ChunkStringToSignTest(
+                AwsSignatureType.HTTP_REQUEST_EVENT,
+                "0c0e3b3bf66b59b976181bd7d401927bbd624107303c713fd1e5f3d3c8dd1b1e"
+            )
+        )
+
+        val epoch = Instant.fromEpochSeconds(123_456_789L, 1234)
+        val prevSignature = "last message sts".encodeToByteArray().sha256().encodeToHex().encodeToByteArray()
+        val chunkBody = "test payload".encodeToByteArray()
+
+        for (test in tests) {
+            val config = AwsSigningConfig {
+                credentialsProvider = Credentials("fake access key", "fake secret key").asStaticProvider()
```

Comment:
**Question**: You don't need these credentials for anything, do you? If not, you can just use `testCredentialsProvider`.